src/System.Net.Sockets/src/System/Net/Sockets/SocketFlags.cs

```diff
@@ -5,78 +5,32 @@
 
 namespace System.Net.Sockets
 {
-    /// <devdoc>
-    ///    <para>
-    ///       Provides constant values for socket messages.
-    ///    </para>
-    /// </devdoc>
-    //UEUE
+    // Provides constant values for socket messages.
 
     [Flags]
     public enum SocketFlags
     {
-        /// <devdoc>
-        ///    <para>
-        ///       Use no flags for this call.
-        ///    </para>
-        /// </devdoc>
+        // Use no flags for this call.
         None = 0x0000,
 
-        /// <devdoc>
-        ///    <para>
-        ///       Process out-of-band data.
-        ///    </para>
-        /// </devdoc>
+        // Process out-of-band data.
         OutOfBand = 0x0001,
 
-        /// <devdoc>
-        ///    <para>
-        ///       Peek at incoming message.
-        ///    </para>
-        /// </devdoc>
+        // Peek at incoming message.
         Peek = 0x0002,
 
-        /// <devdoc>
-        ///    <para>
-        ///       Send without using routing tables.
-        ///    </para>
-        /// </devdoc>
+        // Send without using routing tables.
         DontRoute = 0x0004,
 
-        // see: http://as400bks.rochester.ibm.com/pubs/html/as400/v4r5/ic2978/info/apis/recvms.htm
+        // See: http://as400bks.rochester.ibm.com/pubs/html/as400/v4r5/ic2978/info/apis/recvms.htm
```

Comment:
Maybe we should just remove the comment.
[This article](http://blog.stephencleary.com/2010/06/on-lighter-note-socketflagsmaxiovectorl.html) is pointing an interesting fact about it.